src/dice/ffi/metall.h

```diff
@@ -5,11 +5,13 @@
 #include <stddef.h>
```

Comment:
Possibly unused #include directive
Clang-Tidy: Including 'stdbool. h' has no effect in C++; consider removing it

Reply:
Well this isn't always c++